web/src/components/SocialMediaTooltip.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+import { getSlugFromPath, POIS_ROUTE } from 'api-client'
+import { UiDirectionType } from 'translations'
+
+import { CloseIcon, FacebookIcon, MailSocialIcon, WhatsappIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import { RouteType } from '../routes'
+import Tooltip from './Tooltip'
+
+type SocialMediaTooltipProps = {
+  active: boolean
+  children: ReactNode
+  shareLink: string
+  title: string
+  onClose: () => void
+  flow: 'top' | 'right'
+  direction: UiDirectionType
+  route: RouteType
+}
+
+const SHARE_BUTTON_WIDTH = 50
+const TooltipContainer = styled.div<{
+  flow: 'top' | 'right'
+  active: boolean
+  direction: UiDirectionType
+  additionalPadding: number
+}>`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  padding: 8px;
+  border: 2px solid ${props => props.theme.colors.textDecorationColor};
+  width: max-content;
+  position: absolute;
+  display: flex;
+  z-index: 2000;
+  opacity: 0;
+
+  ${props =>
+    props.flow === 'top' &&
+    css`
+      bottom: ${dimensions.toolbarHeight + props.additionalPadding}px;
+      flex-flow: column-reverse;
+      margin-left: -10px;
+    `};
+
+  ${props =>
+    props.flow === 'right' &&
+    props.direction === 'ltr' &&
+    css`
+      left: ${dimensions.toolbarWidth - SHARE_BUTTON_WIDTH}px;
+    `};
+
+  ${props =>
+    props.flow === 'right' &&
+    props.direction === 'rtl' &&
+    css`
+      right: ${dimensions.toolbarWidth - SHARE_BUTTON_WIDTH}px;
+      transform: scaleX(-1);
```

Comment:
<img width="330" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/8581f74b-7b69-4900-a6d6-150674bb5c58">

Is that just the translation missing, or do we also need to turn the text again?

Reply:
the translation is missing then the text will be rtl as it comes from ar translation

Reply:
RTL text does not have to be inverted, the browser does that automatically. This way, RTL text is the wrong way round as well. Otherwise we would also need to handle this manually for every single text in the app.

Reply:
@steffenkleinle hm i didn't get if i have to change anything here? Its just about missing translations, no?

Reply:
@f1sh1918 yes, the text should not be transformed bc RTL texts are automatically displayed correctly by the browser. By adding the transforming, they are not correctly displayed anymore.

Reply:
fixed this by only rotating the "bubble"